pennylane/resource/error/error.py

```diff
@@ -142,3 +143,25 @@ def get_error(approximate_op: Operator, exact_op: Operator):
         m1 = qml.matrix(exact_op, wire_order=wire_order)
         m2 = qml.matrix(approximate_op, wire_order=wire_order)
         return qml.math.max(qml.math.svd(m1 - m2, compute_uv=False))
+
+
+def _compute_algo_error(tape) -> Dict[str, AlgorithmicError]:
+    """Given a quantum circuit (tape), this function computes the algorithmic error
+    generated by standard PennyLane operations.
+
+    Args:
+        tape (.QuantumTape): The quantum circuit for which we compute errors
+
+    Returns:
+        dict[str->.AlgorithmicError]: dict with error name and combined error as key-value pair
+    """
+
+    op_errors = {}
+    for op in tape.operations:
+        if isinstance(op, ErrorOperation):
+            error_val = op.error()  # get error object from the op
+            error_str = getattr(error_val.__class__, "__name__")  # get name of the class as str
+            error_obj = op_errors.get(error_str, None)  # attempt obtaining error from dict
```

Comment:
Please see [here](https://docs.pennylane.ai/en/stable/development/guide/documentation.html#commenting-code) and my comment above. I think these comments are more of a distraction than help.

Reply:
Thanks, I have updated this.